### PR TITLE
FIX version of packages, fixes #4956

### LINKED ISSUE #4956: @storybook/addon-options@4.0.12 is using @storybook/addons@4.0.8
**Describe the bug**
Is this a mistake? I see there is a `@storybook/addons@4.0.12`.

**To Reproduce**
N/A

**Expected behavior**
`@storybook/addon-options@4.0.12` to use `@storybook/addons@4.0.12`

**Screenshots**
N/A

**Code snippets**
N/A

**System:**
N/A

**Additional context**
N/A

### PR DESCRIPTION
**Pull request by jmfrancois**
Issue: #4956

## What I did

I have updated the dependency

## How to test

- Install this addon and check the dependency is in the same version

### CODE REVIEW

**Comment by shilman** [comment]
Thank you @jmfrancois! This is weird and should be picked up / updated by Lerna automatically. @ndelangen @pksunkara @igor-dv Any ideas why this might fail?

**Comment by pksunkara** [comment]
Looks like it is being picked up correctly as seen [here](https://github.com/storybooks/storybook/blob/next/addons/options/package.json#L23), maybe an exception here.

**Comment by igor-dv** [comment]
I've added an empty PREACT build to `next`, in order to have it working in other branches. Probably need to merge it to `master`.

**Comment by igor-dv** [comment]
this - https://github.com/storybooks/storybook/commit/53c4d36730614f3f8fd2aea01a4e3c72be8e5a4a

**Comment by igor-dv** [comment]
@jmfrancois, can you please merge from `master`?

File: addons/options/package.json [hunk 1]
```diff
@@ -23,7 +23,7 @@
     "prepare": "node ../../scripts/prepare.js"
   },
   "dependencies": {
-    "@storybook/addons": "4.0.8",
+    "@storybook/addons": "4.0.12",
```
Comment (ndelangen) [inline addons/options/package.json]: Why is this an exact dependency?

**Review by ndelangen — APPROVED** [review]

File: addons/options/package.json (hunk 1 above)
Comment (igor-dv) [reply addons/options/package.json]: AFAIK we have exact deps to our pkgs everywhere.

Comment (hulkish) [reply addons/options/package.json]: @igor-dv this is the problem

**Comment by ndelangen** [comment]
I merged in master for you

**Review by ndelangen — APPROVED** [review]

**Pull request merged**